plugins/plugin-bash-like/src/pty/stdio-channel.ts

```diff
@@ -74,23 +74,6 @@ export class StdioChannelWebsocketSide extends EventEmitter implements Channel {
       })
     })
 
-    const self = this
-    const interval = setInterval(function ping() {
-      self.wss['clients'].forEach(function each(ws) {
-        if (ws.isAlive === false) {
-          debugW('killing child process, because client connection did not respond to ping')
-          child.kill()
-          return ws.terminate()
-        }
-
-        // assume it is dead until we get a pong
-        ws.isAlive = false
-        ws.ping(() => {
-          // intentional no-op
-        })
-      })
-    }, pollInterval)
-
```

Comment:
this code is absolutely not dead code. if this is a testament to the caliber of the tool, call me a skeptic?

Reply:
Where is the `interval` used?

Reply:
it's a setInterval, this has global scope

Reply:
But then you don't need the `const`, right?

Reply:
yes, we can eliminate the variable, but not the setInterval call, i believe?

Reply:
Updated!